addons/spreadsheet/static/tests/charts/model/odoo_chart_plugin.test.js

```diff
@@ -819,6 +819,90 @@ test("See records when clicking on a bar chart bar", async () => {
     expect.verifySteps(["load-action", "do-action"]);
 });
 
+test("See records when clicking on a line chart point", async () => {
+    const action = {
```

Comment:
nitpick: I'm not sure everything that in the test is needed, 80 lines to test a click seems a lot. There's probably some fat to trim :)